src/components/styles/NavBar.styled.ts

```diff
@@ -1,20 +1,31 @@
 import styled from 'styled-components'
 
-export const NavContainer = styled.div`
+export const MainContainer = styled.div`
   width: 100vw;
-  height: 6vw;
+  height: 5.5rem;
   display: flex;
-  justify-content: space-evenly;
   ${(props) => `background-color: ${props.theme.palette.common.black};`}
 `
-
 export const BufferContainer = styled.div`
-  height: 30px;
+  width: 2vw;
 `
-
 export const NavBarStyles = styled.div<{ isVisible: boolean }>`
-  width: 65%;
+  width: 60%;
   display: flex;
   justify-content: space-evenly;
+  margin-left: 8%;
+  margin-top: 0.5%;
   top: ${(props) => (props.isVisible ? '0' : '-60px')};
+  @media (max-width: 900px) {
+    margin-left: 4%;
+    margin-top: 3%;
+    width: 80%;
+  }
+  @media (max-width: 650px) {
+    margin-left: 0%;
```

Comment:
if value is 0 no need units i.e. margin-left: 0;

Reply:
fixed in ebef87e